src/java.base/macosx/classes/sun/nio/fs/BsdFileStore.java

```diff
@@ -97,7 +99,8 @@ public boolean supportsFileAttributeView(Class<? extends FileAttributeView> type
                 return true;
             if ("apfs".equals(fstype)) {
                 // fgetxattr broken on APFS prior to 10.14
-                return isOsVersionGte(10, 14);
+                return OperatingSystem.version()
+                        .compareTo(new Version(10, 14)) >= 0;
```

Comment:
The `Version(10, 14)` record should probably be stored in a static constant.

Reply:
Will be removed as it is no longer needed; see PR https://github.com/openjdk/jdk/pull/13628